docs/tutorials/admin/monitoring/index.txt

```diff
@@ -0,0 +1,295 @@
+
+ Monitoring application uses three base entity classes to describe elements of reality: Host, Service Type and Service. 
+
+ * Host is an object describing physical (or virtual) instance of operating system on which GN or GS is running. This object exists only for grouping and is not used directly by monitoring. 
+
+ * Service Type is a description of kind of Service. Depending on service type, different metrics are stored, and different data collection mechanisms are used. Additionally, for system monitoring, it's not conducted directly, but with GeoNode or GeoServer as monitoring agent. That means, no additional software installation is needed to monitor system, but also, hosts that don't have GeoNode or GeoServer installed, won't be monitored. There are four service types:
+   
+   * hostgeonode, hostgeoserver - those types describe system monitoring probes that are running with GeoNode or GeoServer respectively,
+
+   * geonode, geoserver - application-level probes that monitor one specific GeoNode or GeoServer instance.
+
+ * Service describes one specific instance of probe, either host-level or application-level. Service references Host and Service Type. Each service must be named, and name should be system-wide unique.
+
+ As mentioned above, each Service Type keeps a set of metrics, specific for that type. A metric is a description of measured value, for example: number of requests, response size or time, cpu usage, free memory etc. Each Service Type has it's own metrics set. Metric value may be either value counter (like country of user), numeric counter (like number of requests) or rate (like bytes in/out on network interface).
+
+ Besides metric data, monitoring will also store exception information for exceptions that were captured during request handling. 
+
+ Data are collected periodically (at most every 1 minute), aggregated and stored in aggregated form. User can see data from predefined relative periods (last minute, last 10 minutes, last hour, last day, last week).
+
+ User can enable and configure automated checks, which will be run after each collection/aggregation cycle, and will emit notifications if metric values in that run exceed configured thresholds.
+
+
+Installation
+~~~~~~~~~~~~
+
+ * enable ``MONITORING_ENABLED`` flag
+ * ensure that following code is in your settings:
+
+.. code:: python
+
+  MONITORING_ENABLED = True
+  # add following lines to your local settings to enable monitoring
+  if MONITORING_ENABLED:
+      INSTALLED_APPS += ('geonode.contrib.monitoring',)
+      MIDDLEWARE_CLASSES += ('geonode.contrib.monitoring.middleware.MonitoringMiddleware',)
+      MONITORING_SERVICE_NAME = 'local-system-geonode' # or other name for your main geonode instance Service
+      NOTIFICATION_ENABLED = True
+
+* run ``manage.py migrate monitoring`` to apply db schema changes and insert initial data
```

Comment:
On a fresh env, if I run this I get:
CommandError: App 'monitoring' does not have migrations (you cannot selectively sync unmigrated apps)

Reply:
sorry, my bad, my local_settings file was misconfigured